packages/map-template/README.md

```diff
@@ -11,7 +11,7 @@ Install the package:
 In your script:
 
 ```javascript
-import MapsIndoorsMap from '@mapsindoors/map-template/dist/mapsindoors-webcomponent.es.js';
+import MapsIndoorsMap from '@mapsindoors/map-template@stable/dist/mapsindoors-webcomponent.es.js';
```

Comment:
It is not possible to use versions or tags when importing from an already installed package. 

Using a tag should be done at install time.

Reply:
Oh, I wasn't aware of that. I just saw a misalignment with the docs site, and wanted to make sure all the content is correct. I will close this PR.